src/routes/games/games.service.ts

```diff
@@ -23,6 +23,23 @@ export class GamesService {
     });
   }
 
+  findByUserParams(
+    name?: string,
+    languageId?: number,
+    categoryId?: number,
+    limit = 5,
+  ): Promise<Game[] | undefined> {
+    return this.gamesRepository.find({
+      where: {
+        name: ILike(`%${name ?? ''}`),
```

Comment:
Está faltando um '%' depois do nome, pois se for feita a busca por 'car' e só vai encontrar nomes que finalização com 'car' e não qualquer nome que contenha 'car', por exemplo: 'carro'.